dags/glam.py

```diff
@@ -401,17 +362,12 @@
 clients_histogram_aggregates >> clients_histogram_bucket_counts
 clients_histogram_aggregates >> glam_user_counts
 clients_histogram_aggregates >> glam_sample_counts
-
-
 clients_histogram_bucket_counts >> clients_histogram_probe_counts
-clients_histogram_probe_counts >> histogram_percentiles
 
 clients_scalar_aggregates >> glam_user_counts
 glam_user_counts >> extract_counts
 
-
 extract_counts >> extracts_per_channel
 client_scalar_probe_counts >> extracts_per_channel
-scalar_percentiles >> extracts_per_channel
-histogram_percentiles >> extracts_per_channel
+clients_histogram_probe_counts >> extracts_per_channel
```

Comment:
Is this needed as well?
```suggestion
clients_scalar_aggregates >> extracts_per_channel
clients_histogram_probe_counts >> extracts_per_channel
```

Reply:
Thanks, but not really. The dep chain here is:
`clients_scalar_aggregates >> client_scalar_probe_counts`  (L352)
and
`client_scalar_probe_counts >> extracts_per_channel` (L372)